src/App.css

```diff
@@ -13,8 +13,8 @@ svg {
 }
 
 path {
-  stroke-linecap: round;
-  stroke-width: 2;
+  strokeLinecap: round;
+  strokeWidth: 2;
```

Comment:
I've not checked, but it feels like these values should not have change ( I guess you did a global replace for fixing home.tsx )